modern/src/reports/RouteReportPage.js

```diff
@@ -164,6 +176,18 @@ const RouteReportPage = () => {
               )) : (<TableShimmer columns={columns.length + 2} startAction />)}
             </TableBody>
           </Table>
+          {items.length > 100 && (
+            <TablePagination
+              sx={{ position: 'sticky', bottom: 0, left: 0, backgroundColor: 'Background' }}
```

Comment:
I don't like the style here. I think the pagination should be embedded in the layout instead of using "sticky". And ideally it should be in the middle of page at the bottom with proper padding. And you should use class styling.

Also, what is the "Background" color?

Reply:
I cannot seem to understand what "embedded in the layout" exactly means here. Can you please clarify?

Reply:
`position` => `static`